inc/git-updater/namespace.php

```diff
@@ -66,7 +66,7 @@ function update_fair_data( $repo, $repo_api ) : ?WP_Error {
 	foreach ( $versions as $tag => $url ) {
 		// This probably wants to be tied to the commit SHA, so that
 		// if tags are changed, we refresh automatically.
-		$data = generate_artifact_metadata( $did, $url );
+		$data = generate_artifact_metadata( $did, $url, $repo->release_asset );
```

Comment:
How come this uses `$repo->release_asset` rather than `$repo_api->type->release_asset` as you did a few lines above?

Reply:
Fewer characters.

Reply:
In that case, reverse question: How come a few lines above uses `$repo_api->type->release_asset`  rather than `$repo->release_asset` like you're doing here? :joy: